arch/arm/src/samv7/sam_serial_spi.c

```diff
@@ -69,16 +69,16 @@
 
 struct sam_serial_spi_pins_s
 {
-  uint32_t mosi;
-  uint32_t miso;
-  uint32_t sck;
-  uint32_t nss;
+  const uint32_t mosi;
```

Comment:
why add const to integer fields?

Reply:
I can revert. It was just from logical perspective as the values of the pin should not change